compiler/src/iree/compiler/Dialect/Flow/Transforms/Passes.h

```diff
@@ -10,6 +10,7 @@
 #include <functional>
 
 #include "iree/compiler/Dialect/Flow/IR/FlowOps.h"
+#include "iree/compiler/PluginAPI/Client.h"
```

Comment:
The Bazel build is failing on this include: https://github.com/openxla/iree/actions/runs/5664633780/job/15348207870?pr=14493#step:4:225

```
compiler/src/iree/compiler/Dialect/Flow/Transforms/Passes.h:13:10: fatal error: 'iree/compiler/PluginAPI/Client.h' file not found
   13 | #include "iree/compiler/PluginAPI/Client.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

(seems easy to fix by adding a dep in the BUILD.bazel file, unless this is a layering violation and Flow shouldn't depend on PluginAPI?)

Reply:
There is a target that just gets this part so it isn't circular.